saml-based-authentication/README.md

```diff
@@ -0,0 +1,35 @@
+# SAML Integration with Media Shuttle SDK
+
+This documentation specifies steps to integrate SAML with Media Shuttle SDK using [Okta Workforce Identity Cloud](https://www.okta.com) and [AWS Cognito User Pools](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-identity-pools.html).
+
+## Create a SAML app in Okta
+
+- Create an [Okta Workforce Identity](https://www.okta.com/free-trial/workforce-identity/) account and sign in.
+- Create [SAML app integration](https://help.okta.com/en-us/content/topics/apps/apps_app_integration_wizard_saml.htm).
+- Under `GENERAL`, for Single sign on URL, enter `https://yourDomainPrefix.auth.region.amazoncognito.com/saml2/idpresponse`. Replace `yourDomainPrefix` and `region` with the values for your `user pool`.
+- For `Audience URI (SP Entity ID)`, enter `urn:amazon:cognito:sp:yourUserPoolId`. Replace `yourUserPoolId` with your Amazon Cognito user pool ID. Find this value in the Amazon Cognito console on the `General settings` page for your user pool.
+- Under `ATTRIBUTE STATEMENTS (OPTIONAL)`, add a statement with the following information:
+  - For Name, enter the SAML attribute name `http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress`. 
+  - For Value, enter `user.email`.
+- [Assign a user](https://help.okta.com/en-us/content/topics/provisioning/lcm/lcm-assign-app-user.htm) to your Okta application.
+
+## AWS Cognito Configuration
+
+- Get the IdP metadata for your Okta application - On the Sign On tab for your Okta app, find the Identity Provider metadata hyperlink. Right-click the hyperlink, and then copy the URL.
+- Configuration to be done by Signiant SRE in AWS Console.
```

Comment:
Not clear. So there is a part of the config which needs to be done by the customer, and a part which needs to be done by Signiant SRE, correct? If so, then it will be better to clearly identify them in separate sections, not just as a bulletpoint.

Reply:
Segregated the sections in ReadMe.md